### PR TITLE
Feature/text and drive combination

### PR DESCRIPTION
**Pull request by m4rtint**
- It's ready

### CODE REVIEW

File: Assets/_PartyParkade/Script/PowerUp/DriveText/BaseTextAndDriveBehaviour.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+// <copyright file="BaseTextAndDriveBehaviour.cs" company="PERIGON GAMES">
+//     Copyright (c) PERIGON GAMES. 2020 All Rights Reserved
+// </copyright>
+//-----------------------------------------------------------------------
+
+using DG.Tweening;
+using UnityEngine;
+
+[RequireComponent(typeof(CanvasGroup))]
+public class BaseTextAndDriveBehaviour : MonoBehaviour
```
Comment (mikeslgoh) [inline Assets/_PartyParkade/Script/PowerUp/DriveText/BaseTextAndDriveBehaviour.cs:11]: Why do you need a base class?

Comment (m4rtint) [reply Assets/_PartyParkade/Script/PowerUp/DriveText/BaseTextAndDriveBehaviour.cs:11]: They both fade out on initialize, don't want to repeat the same Tweening code

File: Assets/_PartyParkade/Script/PowerUp/DriveText/TextAndDriveBackPanelBehaviour.cs [hunk 2]
```diff
@@ -0,0 +1,3 @@
+﻿public class TextAndDriveBackPanelBehaviour : BaseTextAndDriveBehaviour
```
Comment (Gursaanj) [inline Assets/_PartyParkade/Script/PowerUp/DriveText/TextAndDriveBackPanelBehaviour.cs:1]: Missing header

**Comment by mikeslgoh** [comment]
[Combine the text placement with the prefab (2.0)](https://trello.com/c/87NQRs16/619-combine-the-text-placement-with-the-prefab-20)

**Pull request merged**